packages/wrangler/src/dispatch-namespace.ts

```diff
@@ -181,31 +181,5 @@ export function workerNamespaceCommands(
 					sendMetrics: config.send_metrics,
 				});
 			}
-		)
-		.command(
-			"rename <old-name> <new-name>",
-			"Rename a dispatch namespace",
-			(yargs) => {
-				return yargs
-					.positional("old-name", {
-						describe: "Name of the dispatch namespace",
-						type: "string",
-						demandOption: true,
-					})
-					.positional("new-name", {
-						describe: "New name of the dispatch namespace",
-						type: "string",
-						demandOption: true,
-					});
-			},
-			async (args) => {
-				await printWranglerBanner();
-				const config = readConfig(args.config, args);
-				const accountId = await requireAuth(config);
-				await renameWorkerNamespace(accountId, args.oldName, args.newName);
```

Comment:
Did you need to remove this function too?

Reply:
correct, thank you. removed